src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,48 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || null;
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <Link
+          to={{
+            search: getSearchWith(
+              searchParams, { sex: null },
+            ),
+          }}
+          className={classNames({ 'is-active': !sex })}
+        >
+          All
+        </Link>
+        <Link
+          to={{
+            search: getSearchWith(
+              searchParams, { sex: 'm' },
+            ),
+          }}
+          className={classNames({ 'is-active': sex === 'm' })}
+        >
+          Male
+        </Link>
+        <Link
+          to={{
+            search: getSearchWith(
+              searchParams, { sex: 'f' },
```

Comment:
Consider will be better if you create a enum for magic words and use it everywhere